nucleus/slice.py

```diff
@@ -109,42 +110,12 @@ def items_and_annotations(
             * The other value is a dictionary containing all the annotations for this
                 dataset item, sorted by annotation type.
         """
-        return list(self.items_and_annotation_generator())
-
-    def annotate(
-        self,
```

Comment:
this is useless, better to annotate from dataset

Reply:
agreed